app/src/main/java/org/p2p/wallet/rpc/repository/solana/RpcSolanaRemoteRepository.kt

```diff
@@ -0,0 +1,116 @@
+        val blockHash = blockHashRepository.getRecentBlockhash().recentBlockhash
+        transaction.recentBlockHash = blockHash
+        transaction.sign(signer)
+        val serializedTransaction = transaction.serialize()
+
+        val base64Transaction = Base64Utils.encode(serializedTransaction)
+
+        val params = arrayListOf<Any>().apply {
+            add(base64Transaction)
+            add(RpcSendTransactionConfig())
+        }
+        return api.sendTransaction(
+            RpcRequest(
+                method = "sendTransaction",
+                params = params
+            )
+        ).result
+    }
+
+    override suspend fun getConfirmedSignaturesForAddress(
+        mintLogAccount: PublicKey,
+        limit: Int
+    ): List<SignatureInformationResponse> {
+        val params = arrayListOf<Any>().apply {
+            add(mintLogAccount.toString())
+            add(ConfigObjects.ConfirmedSignFAddr2(limit))
+        }
+        val rawResult = api.getConfirmedSignatureForAddress(
+            RpcRequest(method = "getConfirmedSignaturesForAddress2", params = params)
+        ).result
+
+        val result = arrayListOf<SignatureInformationResponse>().apply {
+            for (item in rawResult) {
+                add(SignatureInformationResponse(item))
+            }
+        }
+        return result
+    }
+
+    override suspend fun getQueryMint(txHash: String): ResponseQueryTxMint {
+        val baseUrl = environmentManager.loadRpcEnvironment().lightNode
+        val params = hashMapOf<String, Any>()
+        params["txHash"] = txHash
+        return api.queryMint(url = baseUrl, rpcRequest = RpcRequest2(method = "ren_queryTx", params = params)).result
+    }
+
+    override suspend fun getQueryBlockState(): ResponseQueryBlockState {
+        val baseUrl = environmentManager.loadRpcEnvironment().lightNode
+        return api.queryBlockState(
+            url = baseUrl,
+            rpcRequest = RpcRequest2(
+                method = "ren_queryBlockState",
+                params = emptyMap()
+            )
+        ).result
+    }
+
+    override suspend fun getQueryConfig(): ResponseQueryConfig {
+        val baseUrl = environmentManager.loadRpcEnvironment().lightNode
+        return api.queryConfig(
+            url = baseUrl,
+            rpcReuest = RpcRequest2(
+                method = "ren_queryConfig",
+                params = emptyMap()
+            )
+        ).result
+    }
+
+    override suspend fun submitTx(
+        hash: String,
+        mintTx: ParamsSubmitMint.MintTransactionInput,
+        selector: String
+    ): ResponseSubmitTxMint {
+        val baseUrl = environmentManager.loadRpcEnvironment().lightNode
+        val submitMint = ParamsSubmitMint(hash, mintTx, selector)
+        val params = hashMapOf<String, Any>()
+        params["tx"] = submitMint
+        return api.submitTx(url = baseUrl, rpcRequest = RpcRequest2(method = "ren_submitTx", params = params)).result
```

Comment:
in the top methods, you are placing each argument on a separate line, maybe you should do this here too